src/commonPrefix/Solution.java

```diff
@@ -0,0 +1,29 @@
+package commonPrefix;
+
+public class Solution {
+    public String longestCommonPrefix(String[] strs) {
+
+        int n = strs.length;
+        System.out.println("valor de n: " + n);
+        char[] prefix = new char[n];
+        for (int i = 0; i < strs.length; i++) {
+            String word = strs[i];
+            char[] letters = word.toCharArray();
+            prefix[i] = letters[i];
```

Comment:
I think that the bug is here. You are using the same index `i`, to traverse two different arrays 1) `strs`, 2) `letters`.